scripts/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -17,6 +18,18 @@ public class PrepareTargets
         [Target(nameof(Init), nameof(RestorePackages))]
         public static BuildTargetResult Prepare(BuildTargetContext c) => c.Success();
 
+        [Target(nameof(CheckPrereqCmakePresent), nameof(CheckPlatformDependencies))]
+        public static BuildTargetResult CheckPrereqs(BuildTargetContext c) => c.Success();
+
+        [Target(nameof(CheckCoreclrPlatformDependencies),  nameof(CheckInstallerBuildPlatformDependencies))]
+        public static BuildTargetResult CheckPlatformDependencies(BuildTargetContext c) => c.Success();
+
+        [Target(nameof(CheckUbuntuCoreclrDependencies),  nameof(CheckCentOSCoreclrDependencies))]
+        public static BuildTargetResult CheckCoreclrPlatformDependencies(BuildTargetContext c) => c.Success();
```

Comment:
For CentOS and Ubuntu we have a declaration of our builds dependencies in the dockerfiles. For mac we don't seem to have any such authoritative information (that I know of). Is libssl the only dependency?

Given that mac does not have a native package manager, checking for installed packages is not as straightforward either. I believe we could use `ld` but will need to ensure this.